core/net/rpl/rpl-mrhof.c

```diff
@@ -118,6 +120,23 @@ reset(rpl_dag_t *dag)
 }
 
 static void
+dao_ack_callback(rpl_parent_t *p, int status)
+{
+  if(status == RPL_DAO_ACK_UNABLE_TO_ACCEPT_ROOT) {
```

Comment:
This is mainly to handle the case then we are not connecting to ROOT. E.g. when root has no longer room for new routes, It is a disaster if all nodes try to switch if we are full at root and many new nodes are trying. They would be better off switching to another DODAG instead - if that is an option. Possibly we could rename it to RPL_DAO_ACK_UNABLE_TO_ADD_ROUTE_AT_ROOT?